src/engine/physics/PhysicsWorld.cpp

```diff
@@ -21,7 +21,7 @@ namespace lei3d
 		m_solver = std::make_unique<btSequentialImpulseConstraintSolver>();
 
 		m_dynamicsWorld = std::make_unique<btDiscreteDynamicsWorld>(m_dispatcher.get(), m_overlappingPairCache.get(), m_solver.get(), m_collisionConfiguration.get());
-		m_dynamicsWorld->setGravity(btVector3(0, -10, 0));
+		m_dynamicsWorld->setGravity(btVector3(0, -40, 0));
```

Comment:
Put this in editor on my most recent commit, so I can just add that in.

Reply:
bet